plugin-viewport/src/main/kotlin/com/mapbox/maps/plugin/viewport/transition/ImmediateViewportTransition.kt

```diff
@@ -37,6 +37,7 @@ internal class ImmediateViewportTransition(delegateProvider: MapDelegateProvider
       cameraPlugin.easeTo(
         cameraOptions,
         mapAnimationOptions {
+          startDelay(0)
```

Comment:
why do we need `startDelay=0`, if not provide delay, isn't it running with zero delay ?

Reply:
Just to make sure, I added this specifically 0. By default it is nul.